### PR TITLE
index: Elaborate on json etc. instead dbus

### PR DESCRIPTION
**Pull request by sils**
DBus is going to be removed. People struggle to find the others. This is
sticked with popular keywords people reportedly search for.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -129,9 +128,10 @@ else than just writing your algorithm!
   Check out :doc:`Writing Bears <Developers/Writing_Bears>` for more
   information on this.
 
-Also, coala provides an external API using the dbus message protocol.
-This allows other applications to easily use the code analysis
-functionalities coala provides in their applications.
+To programmatically access coala's functionality, use ``coala-json`` or
```
Comment (Asnelchristian) [inline index.rst]: i don't think ``programmatically`` exists ?
or maybe you meant ``pragmatically``

**Comment by Asnelchristian** [comment]
ack https://github.com/coala/documentation/pull/279/commits/91c439edcd9ec634983407b05fe752578379e35d

File: index.rst (hunk 1 above)
Comment (sils) [inline index.rst]: according to one user `programmatically` was a specific word he searched this page for

**Comment by sils** [comment]
ack 6afe713 from asnel :)

**Comment by sils** [comment]
@rultor merge

**Comment by Asnelchristian** [comment]
ack https://github.com/coala/documentation/pull/279/commits/6afe71306d61944060b4e2a275f784f6e776ed84

**Pull request merged**